<launch>
    <node name="ser2msg_write_only" pkg="bot_sim" type="ser2msg_write_only" >
        <param name="serial_port" type="string" value="/dev/ttyACM0" />
        <param name="virtual_frame" type="string" value="virtual_frame" />
        <param name="rotbase_frame" type="string" value="rotbase_frame" />
        <param name="gimbal_frame" type="string" value="gimbal_frame" />
        <param name="_3DLidar_frame" type="string" value="aft_mapped" />
        <param name="delta_time" type="double" value="0.04" />
        <param name="vel_topic" type="string" value="/cmd_vel" />
        <param name="topic_name" type="string" value="/angle_topic" />
    </node>
</launch>
